sdk/boms/azure-spring-boot-bom/CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
+
+## 3.2.1 (2021-04-25)
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.3.5.RELEASE](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.3.5.RELEASE/spring-boot-dependencies-2.3.5.RELEASE.pom).
+- Upgrade to [Azure Core 1.13.0](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/CHANGELOG.md#1130-2021-02-05).
+- Upgrade to [Azure Identity 1.2.3](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/identity/azure-identity/CHANGELOG.md#123-2021-02-09).
+- Upgrade to [Azure Key Vault Secret 4.2.5](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md#425-2021-02-11).
+- Upgrade to [Azure MSAL 1.9.1](https://github.com/AzureAD/microsoft-authentication-library-for-java/releases/tag/v1.9.1).
+- Upgrade to [Azure Service Bus JMS 0.0.7](https://search.maven.org/artifact/com.microsoft.azure/azure-servicebus-jms/0.0.7/jar).
+- Upgrade to [Azure Spring Data Cosmos 3.4.0](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md#340-2021-02-09)
+- Upgrade to [Azure Storage Blob 12.10.0](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-blob/CHANGELOG.md#12100-2021-01-14).
+- Upgrade to [Azure Storage File Share 12.8.0](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-file-share/CHANGELOG.md#1280-2021-01-14).
+
+## 3.4.0 (2021-04-23)
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.4.3](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.4.3/spring-boot-dependencies-2.4.3.pom).
+- Upgrade to [Azure Core 1.15.0](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/CHANGELOG.md#1150-2021-04-02).
+- Upgrade to [Azure Identity 1.2.5](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/identity/azure-identity/CHANGELOG.md#release-history).
+- Upgrade to [Azure Key Vault Secret 4.2.7](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md#release-history).
+- Upgrade to [Azure MSAL 1.9.1](https://github.com/AzureAD/microsoft-authentication-library-for-java/releases/tag/v1.9.1).
+- Upgrade to [Azure Service Bus JMS 0.0.7](https://search.maven.org/artifact/com.microsoft.azure/azure-servicebus-jms/0.0.7/jar).
+- Upgrade to [Azure Spring Data Cosmos 3.6.0](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md#360-2021-04-06).
+- Upgrade to [Azure Storage Blob 12.10.2](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-blob/CHANGELOG.md#12102-2021-03-26).
+- Upgrade to [Azure Storage File Share 12.8.0](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-file-share/CHANGELOG.md#1280-2021-01-14).
```

Comment:
This entry already exist in 3.3.0. maybe we should delete it in 3.4.0?

Reply:
Just add this to show a full picture of which dependencies are included.

Reply:
Then it's not change log, it's just a duplication of the content of the pom. 
How about do like this:
1. Rename `### Dependency Upgrades` to `### Dependency Versions`
2. Convert the content to a table.

Reply:
I'd prefer keeping the current format, by following https://github.com/spring-projects/spring-boot/releases.

Reply:
Indeed it's not an upgrade, I can remove them。

Reply:
And I prefer to move the table to readme. 

The table should be like this:
|    | artifact-1 |artifcat-2|
| --| -- | --| 
|  bom:3.6.0| version-1-1| version-2-1| 
|  bom:3.5.0| version-1-2| version-2-1|

Reply:
And link for each version is not necessary. It's OK to add link to each artifact.

Reply:
So you deleted the duplicated items. It's OK for me.